.circleci/config.yml

```diff
@@ -5,9 +5,12 @@ jobs:
       - image: python:3.6
     steps:
       - checkout
+      - run:
+          name: Clone submodules
+          command: git submodule update --init --recursive
       - run:
           name: Install pip dependencies
-          command: pipenv install --dev
+          command: pip install pipenv && pipenv install --dev
```

Comment:
And that? Did the Docker image change?

Reply:
It didn't work previously. I pushed the initial config directly to master.